src/library_idbfs.js

```diff
@@ -16,7 +16,9 @@ addToLibrary({
       if (typeof indexedDB != 'undefined') return indexedDB;
       var ret = null;
       if (typeof window == 'object') ret = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
+#if ASSERTIONS
       assert(ret, 'IDBFS used, but indexedDB not supported');
+#endif
```

Comment:
I think it would be great to get test coverage for these changes.  Perhaps we can add a `_closure` variant to one or more of the existing browser tests?

Reply:
Yes, I'm fine with landing this.  Especially now that it doesn't include the specialHTMLElements thing which was a little confusing to me (since its declared as an array but used like an object).